src/gui/sdlmain.cpp

```diff
@@ -294,6 +302,8 @@ struct SDL_Block {
 		MouseControlType control_choice;
 		bool middle_will_release;
 	} mouse;
+	int  ppscale_x, ppscale_y; /* x and y scales for pixel-perfect     */
+	char dbl_h, dbl_w;         /* double-height and double-width flags */
```

Comment:
`char`?

Reply:
I did not use `bool` because it was not originally available in *C*. Will make them `bool` now.

Reply:
Please describe the bug so I could reproduce it and we're sure we are talking abouth the same issue.